packages/fxa-auth-server/test/local/payments/stripe.js

```diff
@@ -243,6 +389,57 @@ describe('StripeHelper', () => {
         'Customer for email: test@example.com in Stripe has mismatched uid'
       );
     });
+
+    it('returns void if no customers are found', async () => {
+      const listResponse = {
+        autoPagingToArray: sinon.fake.resolves([]),
+      };
+      sandbox.stub(stripeHelper.stripe.customers, 'list').returns(listResponse);
+
+      assert.isEmpty(stripeHelper.fetchCustomer('user123', 'test@example.com'));
+    });
+
+    describe('when a customer has subscriptions and they are more than one page', () => {
+      it('loads all of the subscriptions for the user', async () => {
+        const customer = Object.create(newCustomer);
+        const custSubscription1 = Object.create(subscription1);
+        const custSubscription2 = Object.create(subscription2);
+
+        customer.subscriptions.data = [custSubscription1];
+        customer.subscriptions.has_more = true;
+
+        const listResponse = {
+          autoPagingToArray: sinon.fake.resolves([customer]),
+        };
+        sandbox
+          .stub(stripeHelper.stripe.customers, 'list')
+          .returns(listResponse);
+        sandbox
+          .stub(stripeHelper, 'fetchAllSubscriptionsForCustomer')
+          .resolves([custSubscription2]);
+
+        const result = await stripeHelper.fetchCustomer(
+          'testuid',
+          customer.email
+        );
+
+        assert.isFalse(result.subscriptions.has_more);
+        assert(
+          result.subscriptions.data.length === 2,
+          'both subscriptions should be loaded'
```

Comment:
`assert.strictEqual` is usually more useful in these cases since it'll output the actual value, while the `assert` message is less useful as it doesn't output the actual value.